systems/plants/RigidBodyManipulator.cpp

```diff
@@ -382,8 +391,8 @@ void RigidBodyManipulator::resize(int ndof, int num_featherstone_bodies, int num
   dTdTmult = MatrixXd::Zero(3*num_dof,4);
 
   // preallocate for CMM function
-  Ic.resize(NB); // body spatial inertias
-  dIc.resize(NB); // derivative of body spatial inertias
+  Ic.resize(NB); // composite rigid body inertias
```

Comment:
Will remove these later in favor of Ic_new and dIc_new. Ic_new and dIc_new are std::vectors of matrices with size num_positions, instead of a std::vector of size NB and a std::vector of std::vectors of matrices respectively.